### PR TITLE
perf(collaboration): add grant permission in aad plugin

### PR DESCRIPTION
**Pull request by KennethBWSong**

### CODE REVIEW

File: packages/fx-core/src/plugins/resource/aad/aadAppClient.ts [hunk 1]
```diff
@@ -228,6 +229,29 @@ export class AadAppClient {
...
+    ctx: PluginContext,
+    stage: string,
+    objectId: string,
+    userObjectId: string
+  ): Promise<void> {
+    try {
+      await GraphClient.grantPermission(TokenProvider.token as string, objectId, userObjectId);
+    } catch (error) {
+      if (error?.response?.data?.error.message == Constants.createOwnerDuplicatedMessage) {
+        return;
```
Comment (SLdragon) [inline packages/fx-core/src/plugins/resource/aad/aadAppClient.ts:242]: We can add logs here to inform user that the owner has already been added

File: packages/fx-core/src/plugins/resource/aad/errors.ts [hunk 2]
```diff
@@ -160,6 +165,7 @@ export class GraphClientErrorMessage {
     "Failed to create an application secret in Azure Active Directory.";
   static readonly GetFailed = "Failed to retrieve Azure Active Directory application registration.";
   static readonly CheckPermissionFailed = "Failed to check permission in Azure Active Directory.";
+  static readonly GrantPermissionFailed = "Failed to check permission in Azure Active Directory.";
```
Comment (SLdragon) [inline packages/fx-core/src/plugins/resource/aad/errors.ts:168]: typo: to **grant** permission

File: packages/fx-core/src/plugins/resource/aad/aadAppClient.ts (hunk 1 above)
Comment (KennethBWSong) [reply packages/fx-core/src/plugins/resource/aad/aadAppClient.ts:243]: Fixed. Thx!

File: packages/fx-core/src/plugins/resource/aad/errors.ts (hunk 2 above)
Comment (KennethBWSong) [reply packages/fx-core/src/plugins/resource/aad/errors.ts]: Fixed. Thx!

**Review by SLdragon — APPROVED** [review]

**Pull request merged**